scripts/index.js

```diff
@@ -51,4 +50,15 @@ projects.forEach(function(project) {
   $('#portfolio-pages').append(project.toHtml());
 });
 
+Project.fetchAll = function() {
```

Comment:
You should also add in an if/else conditional statement in this method so that if the data is already present in local storage, you load it from there, and if it's not in local storage, THEN you call the getJSON method with the .then block.